producers/http/handlers_test.go

```diff
@@ -1,5 +1,3 @@
-//+build unit
-
```

Comment:
This should not have been removed - I corrected it in a later commit. I didn't squash it in, in case doing so broke the downstream DC/OS PR.